docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/index.md

```diff
@@ -23,6 +23,7 @@ With data products, you can:
 - **Description;** a description of the data that the data product captures
 - **Owner;** the individual responsible for the data product 
 - **Domain;** the team or business domain that owns the data product
+- **Source Application;** the [source application/s](../organize-data-sources-with-source-applications/index.md) the Data Product is implemented in
```

Comment:
please consider this version, IMO a bit more clear

"Source Application in which the Data Product is implemented"